ivy/functional/frontends/torch/loss_functions.py

```diff
@@ -2,6 +2,43 @@
 import ivy
 
 
+def _get_reduction_func(reduction):    
+    if reduction == 'none':
+        ret = lambda x : x
+    elif reduction == 'mean':
+        ret = ivy.mean
+    elif reduction == 'elementwise_mean':
```

Comment:
just curious, could you explain why did you add `elementwise_mean`? as far as i'm concerned, torch only has 3 `reduction`s

Reply:
`elementwise_mean` is a deprecated option for many changes in torch. 
It seems to be an option that remained as I wrote it referring to the `get_enum` of `torch.nn._reduction.py` among the torch's official code. 
It's an old deprecated setting, so it's okay to delete it. 
I will delete it to reflect the review.
The `elementwise_mean` option will be removed.